tools/accuracy_checker/accuracy_checker/adapters/README.md

```diff
@@ -238,4 +238,7 @@ AccuracyChecker supports following set of adapters:
 * `face_detection_refinement` - converts output of face detection refinement model to `DetectionPrediction` representation. Adapter refines candidates generated in previous stage model.
     * `threshold` - Score threshold to determine as valid face candidate.
 * `attribute_classification` - converts output of attributes classifcation model to `ContainerPrediction` which contains multiple `ClassificationPrediction` for attributes with their scores.
-    * `output_layer_map` - dictionary where keys are output layer names of attribute classification model and values are the names of attributes.
\ No newline at end of file
+    * `output_layer_map` - dictionary where keys are output layer names of attribute classification model and values are the names of attributes.
+* `regression` - converting output of classification model to `RegressionPrediction` representation.
```

Comment:
classification model? not regression?

Reply:
It should be regression. I will fix it in the next commit.